.env.example

```diff
@@ -1 +0,0 @@
-PLASMO_PUBLIC_APP_TYPE=extension
```

Comment:
This one wasn't added by mistake, it's just here so that when new devs copy the `.env` file, they get the default build behavior (building the browser extension).